client/src/app/services/authentication/token-interceptor.spec.ts

```diff
@@ -56,13 +56,13 @@ describe('TokenInterceptor', () => {
 
     // We can assert that the httpRequest’s request.headers object contains a header
     // With the names of ‘Accept’ and ‘Content-Type’. If this is the case then our interceptor has successfully added.
-    expect(httpReq.request.headers.has('Content-Type')).toEqual(true);
+    // expect(httpReq.request.headers.has('Content-Type')).toEqual(true);
```

Comment:
Yes, I just left it here for review, in case this code was absolutely needed, however, if we're sending a form data to the back end, then Content-Type cannot be application/json